db.js

```diff
@@ -1,16 +1,130 @@
 #!/usr/bin/env node
 /* eslint no-unused-expressions: off */
 
-function create() {
+// set up the connection with db
+const pg = require('pg');
+const Sequelize = require('sequelize');
+
+const db = new Sequelize('db_name', 'username', 'password', { // change database name, username, password
+  host: 'host', // change host
+  dialect: 'postgres',
+});
+
+
+
+// definition of the Tweets table
+const Tweet = db.define('tweets', {
+  id: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  text: {
+    type: Sequelize.TEXT,
+  },
+  hashtag: {
+    type: Sequelize.ARRAY(Sequelize.STRING),
+  },
+  timestamp: {
+    type: Sequelize.DATE,
+  },
+  geo: {
+    type: Sequelize.STRING,
+  },
+  location: {
+    type: Sequelize.STRING,
+  },
+});
+
+// definition of the HashtagStats table
+const HashtagStats = db.define('hashtagstats', {
+  country: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  hashtag: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  count: {
+    type: Sequelize.INTEGER,
+    defaultValue: 1
+  },
+});
+
+// returns a list of all the trending hashtags and counties
+function getHashtagStats() {
+// TODO: implement this
+
+}
+
+// inserts and updates hashtag statistics in HashtagStats
+function insertHashtagStats(country, hashtag) {
+  HashtagStats.find({
+      where: {
+        country: country,
+        hashtag: hashtag
+      }
+    }).then((stat) => {
+      if (stat) {
+        stat.increment('count');
+      } else {
+        const data = {
+          country,
+          hashtag,
+        };
+        HashtagStats.create(data).then((row) => {
+          console.log(`Tweet: ${row}`);
+        });
+      }
+    });
+}
```

Comment:
This would work, but I would think about making this so that it only writes to the database every 1000 tweets or so.